From 03aa31c5543913684fe3cdabbc0eafc9c04313a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Thu, 5 Feb 2015 04:36:19 +0100
Subject: [PATCH] hotplug: do not attempt to remove containing xenstore dir
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Organization: Invisible Things Lab
Cc: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>

DomU have no write access to parent directory, so the whole operation
would fail.

Signed-off-by: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>
---
 tools/hotplug/Linux/xen-hotplug-cleanup | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/hotplug/Linux/xen-hotplug-cleanup b/tools/hotplug/Linux/xen-hotplug-cleanup
index fcaa8fc..7aacfc3 100644
--- a/tools/hotplug/Linux/xen-hotplug-cleanup
+++ b/tools/hotplug/Linux/xen-hotplug-cleanup
@@ -32,12 +32,12 @@ else
 fi
 
 # remove device frontend store entries
-xenstore-rm -t \
+xenstore-rm \
   $(xenstore-read "$XENBUS_PATH/frontend" 2>/dev/null) 2>/dev/null || true
 
 # remove device backend store entries
-xenstore-rm -t "$XENBUS_PATH"        2>/dev/null || true
-xenstore-rm -t "error/$XENBUS_PATH"  2>/dev/null || true
+xenstore-rm "$XENBUS_PATH"        2>/dev/null || true
+xenstore-rm "error/$XENBUS_PATH"  2>/dev/null || true
 
 # remove device path from /vm/UUID
 [ "$vm_dev" != "" ] && xenstore-rm -t "$vm_dev" 2>/dev/null || true
-- 
1.8.3.1

